Connectors/Tekla/Speckle.Connector.TeklaShared/Bindings/TeklaBasicConnectorBinding.cs

```diff
@@ -20,23 +19,29 @@ public class TeklaBasicConnectorBinding : IBasicConnectorBinding
   public IBrowserBridge Parent { get; }
   private readonly ILogger<TeklaBasicConnectorBinding> _logger;
   private readonly TSM.Model _model;
+  private readonly ITopLevelExceptionHandler _topLevelExceptionHandler;
 
   public TeklaBasicConnectorBinding(
     IBrowserBridge parent,
     ISpeckleApplication speckleApplication,
     DocumentModelStore store,
     ILogger<TeklaBasicConnectorBinding> logger,
-    IEventAggregator eventAggregator,
-    TSM.Model model
+    TSM.Model model,
+    ITopLevelExceptionHandler topLevelExceptionHandler
   )
   {
     _speckleApplication = speckleApplication;
     _store = store;
     Parent = parent;
     _logger = logger;
     _model = model;
+    _topLevelExceptionHandler = topLevelExceptionHandler;
     Commands = new BasicConnectorBindingCommands(parent);
-    eventAggregator.GetEvent<DocumentStoreChangedEvent>().Subscribe(OnDocumentStoreChangedEvent);
+    _store.DocumentChanged += (_, _) =>
+      _topLevelExceptionHandler.FireAndForget(async () =>
+      {
+        await Commands.NotifyDocumentChanged();
```

Comment:
`OnDocumentStoreChangedEvent` still exists

Reply:
same as arcgis, just forget to remove. now removing